en-US/Docs/IoTCoreShell.md

```diff
@@ -0,0 +1,90 @@
+
+Herein lie the main differences between foreground and background applications:
+
+- **Foreground applications** have a UI. One of these is launched at startup when the device is in headed mode. All foreground apps are registered on the device and the user can switch between foreground apps during device operation.
+
+- **Background applications** have no UI and thus save device resources by turning off the UI stack. Background applications often run continuously from startup and are often used to monitor the device.
+
+## Switching between apps with a Home App
+
+There are two approaches to create a home app for IoT Core which allows you to switch between different foreground applications. Each has an associated sample on GitHub.
+
+The **IoT Home App** ([sample]({{site.baseurl}}/{{page.lang}}/Samples/iothomeapp)) shows you how to create a simple "Home" app that can navigate back and forth between your other apps and the "Home" app.
+
+The **IoT Startup App** ([sample]({{site.baseurl}}/{{page.lang}}/Samples/iotstartapp)) represents a simple startup app that lists the installed apps on your device, then launches one using the PackageManager APIs.
+
+Use either one of these apps as a starting point for your multi-app experience as you see fit.
+
+## Switching between apps with HID Injection Keys
+
+### Return Home
+
+The "GO HOME" key, which returns the default headed app to the foreground, is set to the release of the Windows Button on a keyboard. If you don't have a keyboard on your IoT Device and need to inject low-level keyboard events through [HID Injection]({{site.baseurl}}/{{page.lang}}/Samples/hidinjection), or if you just want to re-map the "GO HOME" functionality to a different key in your app, you can adjust the key value in the registry. For example, to enable pressing the ESCAPE key (0x1B) to "GO HOME", enter the following command in the registry:
+
+``
+“HKLM\Software\Microsoft\Windows NT\CurrentVersion\Winlogon\IoTShellExtension\HotKeys” “HOME” QWORD    0x0000000 0000001B  
+``
+
+As a REG file, this looks as follows:
+
+``
+[HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Winlogon\IoTShellExtension\HotKeys]
+"Home"=hex(b):1B,00,00,00,00,00,00,00
+``
+
+### Switch Between Apps
+
+Alternatively, if you want to switch between your foreground apps, you can set up Alt-Tab (next app) and Shift-Alt-Tab (previous app) functionality in your image by entering the following command in the registry:
+
+``
+“HKLM\Software\Microsoft\Windows NT\CurrentVersion\Winlogon\IoTShellExtension\HotKeys”
+	“PREV”        QWORD       0x00010000 00010009 
+    “NEXT”        QWORD    0x00020000 00050009 
+``
+
+As a REG file, this looks as follows:
+``
+[HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Winlogon\IoTShellExtension\HotKeys]
+"Prev"=hex(b):09,00,01,00,00,00,01,00
+"Next"=hex(b):09,00,05,00,00,00,02,00
+``
+
+### Bit Translation
+
+The above REG file entries decode left to right as follows:

```

Comment:
There is also an optional feature package that enables the Home Windows key and ALT[+SHIFT]+TAB. 
Should call it out here